Sources/PostgresNIO/New/Messages/ReadyForQuery.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 07.01.21.
+//
+
+extension PSQLBackendMessage {
```

Comment:
Ditch Xcode template headers.
```suggestion
extension PSQLBackendMessage {
```